x11-libs/libxrender/libxrender-0.9.10.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Library for the Render Extension to the X11 protocol"
+DESCRIPTION="The Xrender library is designed as a lightweight library \
+interface to the Render extension."
+HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
+COPYRIGHT="2000 SuSE, Inc
+	2001,2003 Keith Packard"
+LICENSE="MIT (no promotion)"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXrender-$portVersion.tar.gz"
+CHECKSUM_SHA256="770527cce42500790433df84ec3521e8bf095dfe5079454a92236494ab296adf"
+SOURCE_FILENAME="libXrender-$portVersion.tar.gz"
```

Comment:
```diff
-SOURCE_FILENAME="libXrender-$portVersion.tar.gz"
```
(not needed, as it's the default value)

BTW, the tar.bz2 is available.
And I've checked that the tar inside the bz2 is identical to the tar inside the gz, so you might wish to siwtch to it:
```diff
-SOURCE_URI="https://www.x.org/releases/individual/lib/libXrender-$portVersion.tar.gz"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXrender-$portVersion.tar.bz2"
-CHECKSUM_SHA256="770527cce42500790433df84ec3521e8bf095dfe5079454a92236494ab296adf"
+CHECKSUM_SHA256="c06d5979f86e64cabbde57c223938db0b939dff49fdb5a793a1d3d0396650949"
```

Reply:
Should we use bz2 over tar.gz? Always used the last ones, but if it's better I can easily switch :)

Reply:
It's not _better_, it's just _smaller_ :)

(BTW, the reason why some people prefer to stick to gz is that some OSes don't have bzip2.)